packages/react/src/components/Card/CardMedia/Controls/SeekButton.js

```diff
@@ -5,13 +5,19 @@ import MediaButton from './MediaButton'
 import { isLarge, media } from '../../../../utils'
 
 const Backward = props => (
-  <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 29' {...props}>
+  <svg
+    xmlns='http://www.w3.org/2000/svg'
+    width='24'
+    height='29'
```

Comment:
We don't need the width/height attributes on the SVGs @Kikobeats

Reply:
removed!